models/job.go

```diff
@@ -419,6 +418,9 @@ type HTTPDependency struct {
 type JobService interface {
 	// Create constructs a Job and commits it to a storage
 	Create(context.Context, NamespaceSpec, JobSpec) (JobSpec, error)
+
+	CreateAndDeployJobSpecifications(context.Context, NamespaceSpec, []JobSpec, writer.LogWriter) (DeploymentID, error)
```

Comment:
@Mryashbhardwaj should we just keep the name `CreateAndDeploy`? Since for refresh and deploy we named it `Refresh` and `Deploy`.

Reply:
yeah we should do that, ill fix this

Reply:
done